### ISSUE TITLE
simple code breaks the build

### ISSUE THREAD

**Issue by yegor256** [objectionary/jeo-maven-plugin#217]
This is my Java:

```java
package foo;
class Foo {
    int bar(double x) {
        if (x > 0.0d) {
            return 5;
        }
        return 8;
    }
}
```

I'm getting:

```
A type incompatibility occurred while executing org.eolang:jeo-maven-plugin:0.1.3:bytecode-to-eo: class org.objectweb.asm.Label cannot be cast to class java.lang.Integer (org.objectweb.asm.Label is in unnamed module of loader org.codehaus.plexus.classworlds.realm.ClassRealm @7c663eaf; java.lang.Integer is in module java.base of loader 'bootstrap')
```

**Comment by volodya-lombrozo** [comment]
@yegor256 Thank you! Could you provide a bit of context? Which java do you use? Do you use Maven to build a project? If so, which Maven configuration you have?

**Comment by yegor256** [comment]
@volodya-lombrozo yes, it's Maven project, with `pom.xml`:

```
$ java -version
openjdk version "20.0.1" 2023-04-18
OpenJDK Runtime Environment Temurin-20.0.1+9 (build 20.0.1+9)
OpenJDK 64-Bit Server VM Temurin-20.0.1+9 (build 20.0.1+9, mixed mode)

$ mvn -version
Apache Maven 3.9.4 (dfbb324ad4a7c8fb0bf182e6d91b0ae20e3d2dd9)
Maven home: /opt/homebrew/Cellar/maven/3.9.4/libexec
Java version: 20.0.1, vendor: Eclipse Adoptium, runtime: /Users/yb/.sdkman/candidates/java/20.0.1-tem
Default locale: en_RU, platform encoding: UTF-8
OS name: "mac os x", version: "14.0", arch: "aarch64", family: "mac"
```

**Comment by 0pdd** [comment]
@yegor256 the puzzle [#226](https://github.com/objectionary/jeo-maven-plugin/issues/226) is still not solved.

**Comment by 0pdd** [comment]
@yegor256 5 puzzles [#232](https://github.com/objectionary/jeo-maven-plugin/issues/232), [#233](https://github.com/objectionary/jeo-maven-plugin/issues/233), [#234](https://github.com/objectionary/jeo-maven-plugin/issues/234), [#235](https://github.com/objectionary/jeo-maven-plugin/issues/235), [#236](https://github.com/objectionary/jeo-maven-plugin/issues/236) are still not solved; solved: [#226](https://github.com/objectionary/jeo-maven-plugin/issues/226).

**Comment by 0pdd** [comment]
@yegor256 6 puzzles [#232](https://github.com/objectionary/jeo-maven-plugin/issues/232), [#233](https://github.com/objectionary/jeo-maven-plugin/issues/233), [#235](https://github.com/objectionary/jeo-maven-plugin/issues/235), [#236](https://github.com/objectionary/jeo-maven-plugin/issues/236), [#240](https://github.com/objectionary/jeo-maven-plugin/issues/240), [#244](https://github.com/objectionary/jeo-maven-plugin/issues/244) are still not solved; solved: [#226](https://github.com/objectionary/jeo-maven-plugin/issues/226), [#234](https://github.com/objectionary/jeo-maven-plugin/issues/234), [#241](https://github.com/objectionary/jeo-maven-plugin/issues/241).

**Comment by 0pdd** [comment]
@yegor256 5 puzzles [#232](https://github.com/objectionary/jeo-maven-plugin/issues/232), [#233](https://github.com/objectionary/jeo-maven-plugin/issues/233), [#235](https://github.com/objectionary/jeo-maven-plugin/issues/235), [#236](https://github.com/objectionary/jeo-maven-plugin/issues/236), [#244](https://github.com/objectionary/jeo-maven-plugin/issues/244) are still not solved; solved: [#226](https://github.com/objectionary/jeo-maven-plugin/issues/226), [#234](https://github.com/objectionary/jeo-maven-plugin/issues/234), [#240](https://github.com/objectionary/jeo-maven-plugin/issues/240), [#241](https://github.com/objectionary/jeo-maven-plugin/issues/241).

**Comment by 0pdd** [comment]
@yegor256 4 puzzles [#232](https://github.com/objectionary/jeo-maven-plugin/issues/232), [#233](https://github.com/objectionary/jeo-maven-plugin/issues/233), [#235](https://github.com/objectionary/jeo-maven-plugin/issues/235), [#244](https://github.com/objectionary/jeo-maven-plugin/issues/244) are still not solved; solved: [#226](https://github.com/objectionary/jeo-maven-plugin/issues/226), [#234](https://github.com/objectionary/jeo-maven-plugin/issues/234), [#236](https://github.com/objectionary/jeo-maven-plugin/issues/236), [#240](https://github.com/objectionary/jeo-maven-plugin/issues/240), [#241](https://github.com/objectionary/jeo-maven-plugin/issues/241).

**Comment by 0pdd** [comment]
@yegor256 3 puzzles [#232](https://github.com/objectionary/jeo-maven-plugin/issues/232), [#233](https://github.com/objectionary/jeo-maven-plugin/issues/233), [#244](https://github.com/objectionary/jeo-maven-plugin/issues/244) are still not solved; solved: [#226](https://github.com/objectionary/jeo-maven-plugin/issues/226), [#234](https://github.com/objectionary/jeo-maven-plugin/issues/234), [#235](https://github.com/objectionary/jeo-maven-plugin/issues/235), [#236](https://github.com/objectionary/jeo-maven-plugin/issues/236), [#240](https://github.com/objectionary/jeo-maven-plugin/issues/240), [#241](https://github.com/objectionary/jeo-maven-plugin/issues/241).

**Comment by 0pdd** [comment]
@yegor256 2 puzzles [#232](https://github.com/objectionary/jeo-maven-plugin/issues/232), [#244](https://github.com/objectionary/jeo-maven-plugin/issues/244) are still not solved; solved: [#226](https://github.com/objectionary/jeo-maven-plugin/issues/226), [#233](https://github.com/objectionary/jeo-maven-plugin/issues/233), [#234](https://github.com/objectionary/jeo-maven-plugin/issues/234), [#235](https://github.com/objectionary/jeo-maven-plugin/issues/235), [#236](https://github.com/objectionary/jeo-maven-plugin/issues/236), [#240](https://github.com/objectionary/jeo-maven-plugin/issues/240), [#241](https://github.com/objectionary/jeo-maven-plugin/issues/241).

**Comment by 0pdd** [comment]
@yegor256 the puzzle [#244](https://github.com/objectionary/jeo-maven-plugin/issues/244) is still not solved; solved: [#226](https://github.com/objectionary/jeo-maven-plugin/issues/226), [#232](https://github.com/objectionary/jeo-maven-plugin/issues/232), [#233](https://github.com/objectionary/jeo-maven-plugin/issues/233), [#234](https://github.com/objectionary/jeo-maven-plugin/issues/234), [#235](https://github.com/objectionary/jeo-maven-plugin/issues/235), [#236](https://github.com/objectionary/jeo-maven-plugin/issues/236), [#240](https://github.com/objectionary/jeo-maven-plugin/issues/240), [#241](https://github.com/objectionary/jeo-maven-plugin/issues/241).

**Comment by 0pdd** [comment]
@yegor256 all 9 puzzles are solved here: [#226](https://github.com/objectionary/jeo-maven-plugin/issues/226), [#232](https://github.com/objectionary/jeo-maven-plugin/issues/232), [#233](https://github.com/objectionary/jeo-maven-plugin/issues/233), [#234](https://github.com/objectionary/jeo-maven-plugin/issues/234), [#235](https://github.com/objectionary/jeo-maven-plugin/issues/235), [#236](https://github.com/objectionary/jeo-maven-plugin/issues/236), [#240](https://github.com/objectionary/jeo-maven-plugin/issues/240), [#241](https://github.com/objectionary/jeo-maven-plugin/issues/241), [#244](https://github.com/objectionary/jeo-maven-plugin/issues/244).
